### DISCUSSION TITLE
Greenhouse Use

### DISCUSSION THREAD

**Discussion by Alexandros89** [PricelessToolkit/SOILSENS-V5W#7] [General]
Hi,
I already uses their predecessor, the mi flora sensor. But I need more for my green house and I think that we could improve their utilize. I need also the possibility to make continuous reading of moisture in determined time, when I'm giving the plant water, the read of the moisture should be continuous.
Should it be possible?
I don't understand also the difference between the 2 model. One is WiFi and one is Lora?

**Comment by PricelessToolkit** [comment]
Hi, one is "WIFI and ESP-NOW" and the other one is "wired". The measurement cycle is determined by an external hardware timer. 1H 30min 15min .....

**Reply by Alexandros89** [reply]
The WiFi model can be programmed also with esphome instead of Arduino?

**Reply by PricelessToolkit** [reply]
Maybe I didn’t try it. The module is not in sleep mode, it is powered on by an external timer. With ESPHome, you would need to toggle a special pin after sending data to power off the module. If that’s possible, then yes.
